infrastructure/templates/public-api/components/postgresqlDatabase.bicep

```diff
@@ -53,18 +53,46 @@ param geoRedundantBackup string = 'Disabled'
 @description('The name of the Key Vault to store the connection strings')
 param keyVaultName string
 
+@description('Specifies the VNet id')
+param vNetId string
+
+@description('Specifies the subnet id')
+param subnetId string
+
+param databaseNames array
+param firewallRules array
+
 //Passed in Tags
 param tagValues object
 
 // Variables and created data
-var databaseName = '${resourcePrefix}-psql-${serverName}'
-var connectionStringSecretName = '${resourcePrefix}-sql-${serverName}-connectionString'
+var databaseServerName = empty(serverName)
+  ? '${resourcePrefix}-psql'
+  : '${resourcePrefix}-psql-${serverName}'
+
+var connectionStringSecretName = '${databaseServerName}-connectionString'
```

Comment:
I think we can tackle this one when it's in the EES Dev Resource Group and we can start to play with resources in place.  Alternatively we can continue on with this in a Hive test Resource Group as I have been doing so far, but I'd do it as a follow-up PR as I think we need to prove that we're able to get this work into the proper EES Resource Groups sooner rather than later.